packages/core-data/src/controls.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { default as triggerApiFetch } from '@wordpress/api-fetch';
+import { select as selectData } from '@wordpress/data';
+
+/**
+ * Trigger an API Fetch request.
+ *
+ * @param {Object} request API Fetch Request Object.
+ * @return {Object} control descriptor.
+ */
+export function apiFetch( request ) {
+	return {
+		type: 'API_FETCH',
+		request,
+	};
+}
+
+/**
+ * Calls a selector using the current state.
+ * @param {string} selectorName Selector name.
+ * @param  {Array} args         Selector arguments.
+ *
+ * @return {Object} control descriptor.
+ */
+export function select( selectorName, ...args ) {
+	return {
+		type: 'SELECT',
+		selectorName,
+		args,
```

Comment:
It seems weird that this isn't a spread here but is above… does that mean this is doing: `args: …args`? Sometimes I get hazy with ES6 shorthands.

Reply:
In this particular case, it's better to keep it as a separate `args` property, to avoid ambiguity with the others keys (type, selectorName)